TokenCacheMigration/CommonCacheMsalV3/Program.cs

```diff
@@ -15,6 +15,7 @@ static void Main(string[] args)
         {
             Console.WriteLine(Path.GetFileNameWithoutExtension(Assembly.GetEntryAssembly().Location));
             DoIt().Wait();
+            Console.ReadLine();
```

Comment:
the drawback of adding the Readline is that when you run the .bat file which is in the TokenCacheMigration folder, you need to press enter each time ...

Reply:
@jmprieur I see. I removed it. thanks.